capstone/src/main/webapp/youtubeScript.js

```diff
@@ -65,20 +87,24 @@ async function inputCommentsToPerspective(comments) {
       return;
   }
   const attributeScoresPromises = [];
-  for (const item in comments.items) {
-    const perspectiveScore = callPerspective(comments.items[item].snippet.topLevelComment.snippet.textOriginal, langElement.value, requestedAttributes);
-    attributeScoresPromises.push(perspectiveScore);
+  for (const comments in commentsList) {
+    for (const item in commentsList[comments].items) {
+      const perspectiveScore = await callPerspective(commentsList[comments].items[item].snippet.topLevelComment.snippet.textOriginal, langElement.value, requestedAttributes);
+      attributeScoresPromises.push(perspectiveScore);
+    }
   }
   await Promise.all(attributeScoresPromises).then(resolvedAttributeScores => {
-    const attributeAverages = getAttributeAverages(resolvedAttributeScores, comments.items.length);
+    const attributeTotals = getAttributeTotals(resolvedAttributeScores);
+    const attributeAverages = getAttributeAverages(attributeTotals, commentsList);
     loadChartsApi(attributeAverages);
+    perspectiveToxicityScale(attributeAverages);
```

Comment:
When I run this code, I'm getting the error that this method is undefined, and I don't see its implementation in this file. Is it maybe in another branch?

Reply:
I'm unsure how that got there. That function is implemented in branch 21.